Packs/PrismaCloud/Integrations/RedLock/RedLock.yml

```diff
@@ -54,6 +54,22 @@ script:
   type: python
   subtype: python2
   commands:
+  - name: redlock-get-RQL-response
+    description: Run RQL query on Prisma Cloud
+    arguments:
+    - name: rql  
+      required: true
+      default: false
+      isArray: false
+      description: the RQL query to run
+  - name: redlock-remediate-alerts
+    description: Remediate alerts in Prisma Cloud
+    arguments:
+    - name: alert-id
+      required: true
+      default: false
+      isArray: false
+      description: The alert id to remediate
```

Comment:
add description on how can we retrieve this alert id, from which command in the integration

Reply:
added